### DISCUSSION TITLE
Getting error `Uncaught TypeError: Cannot read properties of undefined (reading 'emit')`

### DISCUSSION THREAD

**Discussion by OZcomingFRoo** [vueup/vue-quill#217] [General]
I use Vue 3 and add a simple quill component,

And I get this error no matter what I do:
![image](https://user-images.githubusercontent.com/32063353/178126179-a59520e8-ccd8-491f-9b4b-3c656750e7b3.png)

This is how I code the quill component in my parent component.
![image](https://user-images.githubusercontent.com/32063353/178126202-d2ddabee-8f3c-463e-a5c5-fa90a65b98dc.png)

Is this a new error from the beta version?
This didn't happen before.

What am I missing?

Edit:
Mind that I even tried to remove `v-model:content="tutor.description"` that was highlighted.
And still the same error.

**Comment by mJeromeDiaz** [comment]
Not a QuillEditor error
You want to create QuillEditor before your data is mounted.

just put v-if="tutor.description !== undefined"

**Comment by eigriffiths** [comment]
Did you ever find the answer to this issue? I have the exact same error...
